Makefile

```
@@ -67,8 +67,8 @@ uv-venv:
  # make test-python will produce html coverage by default. Run with `make test-python report=xml` to produce xml report.
 .PHONY: test-python
 test-python: uv-venv
-	@uv sync
-	@uv run coverage run --source=kubeflow.trainer.backends.kubernetes.backend,kubeflow.trainer.utils.utils -m pytest ./kubeflow/trainer/backends/kubernetes/backend_test.py
+	@uv sync --group dev
```

`uv sync` includes dev group by default https://docs.astral.sh/uv/concepts/projects/dependencies/#default-groups
```suggestion
	@uv sync
```